crawler.py

```diff
@@ -190,6 +190,10 @@ def run_all(driver):
 # Remove a prefix from the path where .gcda files are stored
 os.environ['GCOV_PREFIX_STRIP'] = str(strip_count)
 
+# Environment variable for JS engine to emit JS coverage information
+jsvm_dir = tempfile.mkdtemp()
+os.environ['JS_CODE_COVERAGE_OUTPUT_DIR'] = self.jsvm_dir
```

Comment:
This is not in a class, so there should be no `self`

Reply:
Can you move it either after setting GCOV_PREFIX or before setting GCOV_PREFIX_STRIP?